bindings/wasm/Cargo.toml

```diff
@@ -24,20 +24,20 @@ iota-sdk-bindings-core = { path = "../core", default-features = false, features
 ] }
 
 console_error_panic_hook = { version = "0.1.7", default-features = false }
-js-sys = { version = "0.3.64", default-features = false, features = [] }
+js-sys = { version = "0.3.66", default-features = false, features = [] }
 log = { version = "0.4.20", default-features = false }
-serde_json = { version = "1.0.107", default-features = false }
-tokio = { version = "1.33.0", default-features = false, features = ["sync"] }
-wasm-bindgen = { version = "0.2.87", default-features = false, features = [
+serde_json = { version = "1.0.108", default-features = false }
+tokio = { version = "1.34.0", default-features = false, features = ["sync"] }
+wasm-bindgen = { version = "0.2.89", default-features = false, features = [
```

Comment:
Also update in the wasm .yml files for the CI

Reply:
Good catch

Reply:
Done